_episodes/03-configuration.md

```diff
@@ -236,10 +241,10 @@ information about ``drs``, you can visit the ESMValTool documentation on
 > ## Explain the default drs (if working on local machine)
 >
 > 1. In the previous exercise, we set the `drs` of CMIP5 data to `default`.
-> Can you explain why?
->
+>    Can you explain why?
+
```

Comment:
Doesn't this white line break the block?

Reply:
Oops. Well spotted.